packages/grid/x-data-grid/src/styles/styled.ts

```diff
@@ -0,0 +1,6 @@
+import { createStyled } from '@mui/system';
```

Comment:
I think this file could go in `src/utils` folder as we have most of the other utilities there.